crates/biome_formatter/CONTRIBUTING.md

```diff
@@ -161,11 +161,11 @@ where
 </details>
 
 
-Then, you'll have to create three types:
+Then, you'll have to create four types:
 1. `HtmlCommentStyle`
-1. `HtmlFormatContext`
-1. `FormatHtmlSyntaxNode` 
-1. `HtmlLanguage`
+2. `HtmlFormatContext`
+3. `FormatHtmlSyntaxNode` 
+4. `HtmlLanguage`
```

Comment:
nit: Actually, if you prefix these with `1.` it automatically numbers the items correctly when it renders.

Reply:
Ah, gotcha! Thanks for that. I wasn't aware of that, and I was only looking at it from the pure markdown, so I totally missed this